megatron/megatron/model/transformer.py

```diff
@@ -618,15 +618,15 @@ def __init__(self, config, layer_number,
                 query_projection_size,
                 config=config,
                 init_method=config.init_method,
-                bias=config.add_bias_linear,
+                bias=(args.add_bias_linear_qkv or args.add_bias_linear),
```

Comment:
We should also support the new `config.add_bias_linear` instead of using the `args.add_bias_linear` directly since we will sync with the upstream.